lib/screens/profile_screens/security/security_screen.dart

```diff
@@ -82,18 +82,17 @@ class SecurityScreen extends StatelessWidget {
                           );
                         },
                       ),
-                      if (state.shouldShowExportRecoveryPhrase)
+                      // ignore: prefer_if_elements_to_conditional_expressions
+                      true ?
```

Comment:
@gguij004  make a release note to set this value back to state.shouldShowExportRecoveryPhrase
This code is only used for testing as is.